bugsnag-android-core/src/androidTest/java/com/bugsnag/android/ReportTest.java

```diff
@@ -54,7 +54,7 @@ public void testModifyingGroupingHash() throws JSONException, IOException {
     public void testModifyReportDetails() throws Exception {
         report.setApiKey("custom-api-key");
         report.getNotifier().setName("React Native");
-        report.getNotifier().setURL("https://bugsnag.com/reactnative");
+        report.getNotifier().setUrl("https://bugsnag.com/reactnative");
```

Comment:
related?

Reply:
Our code style for JVM languages prefers that acronyms have only the first letter capitalised - we couldn't change this for the `Notifier` class without making a breaking change as these methods were public.

Reply:
I just didn't see the update to this method's definition in this branch… or is this method synthesized on the `Notifier` class because it's now written in kotlin?

Reply:
Yep! `Notifier` has a `url` property, which when compiled down to Java automatically generates a getter + setter:
https://github.com/bugsnag/bugsnag-android/pull/622/files#diff-2a2f9b5da9e45dce6d45a9b80d5d2253R12